### PR TITLE
[3.x] Document editor settings in the class reference

### PR DESCRIPTION
**Pull request by Calinou**
`3.x` version of https://github.com/godotengine/godot/pull/63870.

~~Marked as draft as https://github.com/godotengine/godot/pull/64895 must be merged first for CI to pass.~~
**Edit:** No longer draft.

### CODE REVIEW

File: doc/classes/EditorSettings.xml [hunk 1]
```diff
@@ -157,430 +157,691 @@
...
 		<member name="text_editor/theme/color_theme" type="String" setter="" getter="">
+			The syntax theme to use in the script editor.
+			You can save your own syntax theme from your current settings by using [b]File &gt; Theme &gt; Save As...[/b] at the top of the script editor. The syntax theme will then be available locally in the list of color themes.
+			You can find additional syntax themes to install in the [url=https://github.com/godotengine/godot-syntax-themes]godot-syntax-themes[/url] repository.
 		</member>
 		<member name="text_editor/theme/line_spacing" type="int" setter="" getter="">
+			The space to add between lines (in pixels). Greater line spacing can help improve readability at the cost of displaying fewer lines on screen.
 		</member>
 		<member name="text_editor/tools/create_signal_callbacks" type="bool" setter="" getter="">
 		</member>
```
Comment (Calinou) [inline doc/classes/EditorSettings.xml:841]: This setting is unused, so I made another PR to remove it: https://github.com/godotengine/godot/pull/64892

**Comment by akien-mga** [comment]
Is this still a draft?

**Review by akien-mga — APPROVED** [review]
I didn't proofread in depth, but should be good to merge if it's ready, and any typos can be fixed up later on.

**Comment by Calinou** [comment]
> Is this still a draft?

I forgot to mark this as ready for review – it should be OK to merge now.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**